src/link/link.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { TestBed, ComponentFixture } from "@angular/core/testing";
+import { Component, Input, DebugElement } from "@angular/core";
+import { By } from "@angular/platform-browser";
+
+import { Link } from "./link.directive";
+
+describe("Link", () => {
+	it("should create a Link", () => {
+		TestBed.configureTestingModule({
+			declarations: [TestLinkComponent, Link]
+		});
+
+		let fixture: ComponentFixture<TestLinkComponent> = TestBed.createComponent(TestLinkComponent);
+		let component: TestLinkComponent = fixture.componentInstance;
+		fixture.detectChanges();
+
+		expect(component).toBeTruthy();
+		const directiveEl = fixture.debugElement.query(By.directive(Link));
+		expect(directiveEl).not.toBeNull();
+		expect(directiveEl.attributes["aria-disabled"]).toBe(null);
```

Comment:
It looks like this test is failing. `aria-disabled` doesn't really have to be `null`, it just has to be `falsy`, no?
```suggestion
		expect(directiveEl.attributes["aria-disabled"]).toBeFalsy();
```

Reply:
On a second look, setting `attr.aria-disabled` might have to require some additional logic to mask the value.
Test actually seems to compare a string `'false'` and not the boolean value, and a non-empty string is _truthy_.
It seems to be working fine in other places though, so the check may have to be:
```suggestion
		expect(directiveEl.attributes["aria-disabled"]).toBe("false");
```

Reply:
I'd expect `.toBe(null)` and `.toBeFalsy()` to be equivalent in this case ... falsy attributes are generally expected to be `null`ed out of the DOM anyway.

Reply:
@cal-smith Please see question above. Thanks :-)